### PR TITLE
Reduce flakiness of spotlight tests

### PR DESCRIPTION
**Pull request by justjanne**
Type: Task
Fixes: https://github.com/vector-im/element-web/issues/23155
Fixes: https://github.com/vector-im/element-web/issues/23168

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Review by weeman1337 — APPROVED** [review]
Thanks for fixing the test :+1:

**Review by turt2live — APPROVED** [review]
haven't looked, but approving to clear codeowners review (assuming @weeman1337's review is suitable)

**Pull request merged**
